set.mm

```diff
@@ -643200,6 +643206,781 @@ base set if and only if the neighborhoods (convergents) of every point
+    2nsgsimpgd.4 $e |- ( ph -> -. { .0. } = B ) $.
+    2nsgsimpgd.5 $e |- ( ( ph /\ x e. ( NrmSGrp ` G ) ) ->
+      ( x = { .0. } \/ x = B ) ) $.
+    $( If any normal subgroup of a nontrivial group is either the trivial
+       subgroup or the whole group, the group is simple.  (Contributed by Rohan
+       Ridenour, 3-Aug-2023.) $)
+    2nsgsimpgd $p |- ( ph -> G e. SimpGrp ) $=
+      ( wcel wa wceq adantl simpr syl adantr eqeltrd adantlr cvv cfv csn cpr cv
+      cnsg c2o cen wo elprg mpbird cgrp 0nsg nsgid elpri mpjaodan impbida eqrdv
+      wb snex a1i cbs fvexi enpr2d eqbrtrd issimpgd ) ADHADUEUAZEUBZCUCZUFUGABV
+      FVHABUDZVFKZVIVHKZAVJLVKVIVGMZVICMZUHZJVJVKVNURAVIVGCVFUINUJAVKLVLVJVMAVL
+      VJVKAVLLVIVGVFAVLOAVGVFKZVLADUKKZVOHDEGULPQRSAVMVJVKAVMLVICVFAVMOACVFKZVM
+      AVPVQHCDFUMPQRSVKVNAVIVGCUNNUOUPUQAVGCTTVGTKAEUSUTCTKACDVAFVBUTIVCVDVE $.
+  $}
+
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Classification of abelian simple groups
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  ${
+    $d ph n $.  $d A n $.
+    1gengend.1 $e |- B = ( Base ` G ) $.
+    1gengend.2 $e |- .x. = ( .g ` G ) $.
+    1gengend.3 $e |- F = ( n e. ZZ |-> ( n .x. A ) ) $.
+    1gengend.4 $e |- ( ph -> A e. B ) $.
+    $( The set of multiples of an element of a group includes that element.
+       (Contributed by Rohan Ridenour, 3-Aug-2023.) $)
+    1gengend $p |- ( ph -> A e. ran F ) $=
+      ( cz cv co c1 1zzd wceq wa simpr oveq1d wcel adantr mulg1 syl rr-elrnmptd
+      eqtr2d ) AELEMZBDNZOBFCJAPKAUGOQZRZUHOBDNZBUJUGOBDAUISTUJBCUAZUKBQAULUIKU
+      BCDGBHIUCUDUFUE $.
+  $}
+
+  ${
+    $d a n $.  $d G n $.  $d .x. n $.  $d B n $.  $d A n $.
+    1gensubgd.1 $e |- B = ( Base ` G ) $.
+    1gensubgd.2 $e |- .x. = ( .g ` G ) $.
+    1gensubgd.3 $e |- F = ( n e. ZZ |-> ( n .x. A ) ) $.
+    1gensubgd.4 $e |- ( ph -> G e. Grp ) $.
+    1gensubgd.5 $e |- ( ph -> A e. B ) $.
+    $( The set of multiples of an element of a group is a subgroup.
+       (Contributed by Rohan Ridenour, 3-Aug-2023.) $)
+    1gensubgd $p |- ( ph -> ran F e. ( SubGrp ` G ) ) $=
+      ( crn csubg cfv wcel cgrp wa cycsubgcl syl2anc simpld ) AFMZGNOPZBUBPZAGQ
+      PBCPUCUDRKLEBDFGCHIJSTUA $.
+  $}
+
+  ${
+    $d .x. m n $.  $d A n m $.  $d G m $.  $d B m $.
+    1gensubg2d.1 $e |- B = ( Base ` G ) $.
+    1gensubg2d.2 $e |- .x. = ( .g ` G ) $.
+    1gensubg2d.3 $e |- F = ( n e. ZZ |-> ( n .x. A ) ) $.
+    1gensubg2d.4 $e |- ( ph -> G e. Grp ) $.
+    1gensubg2d.5 $e |- ( ph -> A e. B ) $.
+    $( The set of multiples of an element of a group is a subgroup.  Fewer
+       distinct variable conditions than ~ 1gensubgd .  (Contributed by Rohan
+       Ridenour, 3-Aug-2023.) $)
+    1gensubg2d $p |- ( ph -> ran F e. ( SubGrp ` G ) ) $=
+      ( vm cz cv co cmpt oveq1 cbvmptv eqtri 1gensubgd ) ABCDMFGHIFENEOZBDPZQMN
+      MOZBDPZQJEMNUCUEUBUDBDRSTKLUA $.
+  $}
+
```

Comment:
The previous theorems are general theorems about groups and therefore should be moved to section  "Groups".

Reply:
~ 1gengend (now ~ cycsubggend) depends on ~ rr-rspce. The obvious unprefixed name for that theorem is "rspce", but unfortunately ~ rspce already exists and is somewhat different. I'm not sure how to resolve the naming conflict, since ~ rr-rspce is more basic than most of the rspce-type theorems so it seems to be that it should have the name "rspce", but that raises the question of what the theorem now named ~ rspce should be named. (~ elrnmptd also already exists in Glauco Silprandi's mathbox, but ~ rr-elrnmptd is a trivial consequence of that and ~ rr-rspce, so the obvious solution is to move ~ elrnmptd to after ~ elrnmpt and delete ~ rr-elrnmptd.)

Reply:
OK, then this can be adjusted later.